Packs/FortinetFortiwebVM/Integrations/FortinetFortiwebVM/FortinetFortiwebVM.yml

```diff
@@ -0,0 +1,2407 @@
+  required: true
+  options:
+  - V1
+  - V2
+- defaultvalue: 'false'
+  display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- defaultvalue: 'false'
+  display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+description: "Fortiweb VM integration allows to manage WAF policies and block cookies, URLs, and hostnames."
+display: Fortiweb VM
+name: fortiweb_vm
+script:
+  commands:
+  - arguments:
+    - description: Protected Hostname Group Name.
+      isArray: false
+      name: name
+      required: true
+    - auto: PREDEFINED
+      defaultValue: Allow
+      description: "Select whether to accept or deny HTTP requests whose Host: field does not match any of the host definitions that you will add to this protected hosts group."
+      isArray: false
+      name: default_action
+      predefined:
+      - Allow
+      - Deny (no log)
+      - Deny
+      required: false
+    deprecated: false
+    description: Create Protected Hostname Group.
+    execution: false
+    name: fortiwebvm-protected-hostname-group-create
+  - arguments:
+    - description: Protected Hostname Group Name.
+      isArray: false
+      name: name
+      required: true
+    - auto: PREDEFINED
+      description: "Select whether to accept or deny HTTP requests whose Host: field does not match any of the host definitions that you will add to this protected hosts group."
+      isArray: false
+      name: default_action
+      predefined:
+      - Allow
+      - Deny (no log)
+      - Deny
+      required: false
+    deprecated: false
+    description: Update Protected Hostname Group.
+    execution: false
+    name: fortiwebvm-protected-hostname-group-update
+  - arguments:
+    - description: Protected Hostname Group Name.
+      isArray: false
+      name: name
+      required: false
+    - description: The page number of the results to retrieve.
+      isArray: false
+      name: page
+      required: false
+    - description: The maximum number of records to retrieve per page.
+      isArray: false
+      name: page_size
+      required: false
+    - defaultValue: '50'
+      description: The maximum number of records to retrieve.
+      isArray: false
+      name: limit
+      required: false
+    deprecated: false
+    description: List the Protected Hostname groups.
+    execution: false
+    name: fortiwebvm-protected-hostname-group-list
+    outputs:
+    - contextPath: FortiwebVM.ProtectedHostnameGroup.id
+      description: Protected Hostname Group ID.
+      type: String
+    - contextPath: FortiwebVM.ProtectedHostnameGroup.default_action
+      description: Protected Hostname Group Action.
+      type: Number
+    - contextPath: FortiwebVM.ProtectedHostnameGroup.protected_hostname_count
+      description: Protected Hostname Group members number.
+      type: Number
+    - contextPath: FortiwebVM.ProtectedHostnameGroup.can_delete
+      description: Can deleted flag. Supports API version 1 only.
```

Comment:
Don't understand this description.  Should it be: "Whether the protected hostname group can be deleted?"

Reply:
yes :)